go.mod298498081.tmp

```diff
@@ -0,0 +1,31 @@
+module sigs.k8s.io/cluster-api-provider-azure
```

Comment:
Not sure this should be checked in? Any idea where this file came from?

Reply:
That's vscode

Reply:
I removed it